include/openthread/thread_ftd.h

```diff
@@ -368,7 +368,7 @@ uint32_t otThreadGetContextIdReuseDelay(otInstance *aInstance);
 void otThreadSetContextIdReuseDelay(otInstance *aInstance, uint32_t aDelay);
 
 /**
- * Get the NETWORK_ID_TIMEOUT parameter used in the Router role.
+ * Get the NETWORK_ID_TIMEOUT parameter.
```

Comment:
Since you're editing an API file you'll need to increment the API version in instance.h